filament/src/materials/sao.mat

```diff
@@ -68,18 +72,10 @@ fragment {
         return a * a;
     }
 
-    // remaps to -1 and 1, repeating
-    float reduce(highp float x) {
-        return fract(0.5 * x + 0.5) * 2.0 - 1.0;
-    }
-
-    // very crude and fast sin/cos approximation
-    vec2 fast_cossin(highp float x) {
-        x *= 1.0/PI;
-        vec2 a = vec2(reduce(x + 0.5), reduce(x));
-        vec2 xn = sq(a * 2.0 + 1.0) - 1.0;
-        vec2 xp = 1.0 - sq(a * 2.0 - 1.0);
-        return vec2(a.x < 0.0 ? xn.x : xp.x, a.y < 0.0 ? xn.y : xp.y);
+    vec2 pack(highp float depth) {
+        mediump float z = clamp(depth * materialParams.invFarPlane, 0.0, 1.0);
```

Comment:
We're mediump by default

Reply:
actually, thinking more about it, I think this needs to be highp -- good catch :-)